### PR TITLE
improve FieldParser converters

### PR DESCRIPTION
**Pull request by JoeCqupt**
### Motivation

improve FieldParser converters, let them keep the same code style

### Modifications

update FieldParser converters

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

### Documentation

Check the box below or label this PR directly (if you have committer privilege).

Need to update docs? 
  
- [x] `no-need-doc` 
  
just  code cleanup

### CODE REVIEW

**Comment by JoeCqupt** [comment]
@Jason918 @eolivelli @merlimat  PTAL,thanks!

File: pulsar-common/src/main/java/org/apache/pulsar/common/util/FieldParser.java [hunk 1]
```diff
@@ -274,7 +274,12 @@ public static Integer stringToInteger(String val) {
      * @return The converted Long value.
      */
     public static Long stringToLong(String val) {
-        return Long.valueOf(trim(val));
+        String v = trim(val);
+        if (io.netty.util.internal.StringUtil.isNullOrEmpty(v)) {
```
Comment (codelipenghui) [inline pulsar-common/src/main/java/org/apache/pulsar/common/util/FieldParser.java:278]:
```suggestion
        if (StringUtils.isEmpty(v)) {
```

File: pulsar-common/src/main/java/org/apache/pulsar/common/util/FieldParser.java [hunk 2]
```diff
@@ -301,7 +306,12 @@ public static Double stringToDouble(String val) {
      * @return The converted Double value.
      */
     public static Float stringToFloat(String val) {
-        return Float.valueOf(trim(val));
+        String v = trim(val);
+        if (io.netty.util.internal.StringUtil.isNullOrEmpty(v)) {
```
Comment (codelipenghui) [inline pulsar-common/src/main/java/org/apache/pulsar/common/util/FieldParser.java:310]:
```suggestion
        if (StringUtils.isEmpty(v)) {
```

File: pulsar-common/src/main/java/org/apache/pulsar/common/util/FieldParser.java [hunk 3]
```diff
@@ -384,7 +394,12 @@ public static String booleanToString(Boolean value) {
      * @return The converted Boolean value.
      */
     public static Boolean stringToBoolean(String value) {
-        return Boolean.valueOf(value);
+        String v = trim(value);
+        if (io.netty.util.internal.StringUtil.isNullOrEmpty(v)) {
```
Comment (codelipenghui) [inline pulsar-common/src/main/java/org/apache/pulsar/common/util/FieldParser.java:398]:
```suggestion
        if (StringUtils.isEmpty(v)) {
```

**Review by codelipenghui — APPROVED** [review]
LGTM, just left some minor comments.

**Review by Jason918 — APPROVED** [review]

**Pull request closed without merging**